worlds/yugioh06/Rules.py

```diff
@@ -0,0 +1,876 @@
+from worlds.generic.Rules import add_rule
+from .Fusions import count_has_materials
+
+
+def set_rules(world):
+    player = world.player
+    world = world.multiworld
```

Comment:
Heavily recommend against doing `world = world.multiworld`. Would greatly prefer if you did `multiworld = world.multiworld`